projects/sound-open-firmware/fuzz-ipc.options

```diff
@@ -0,0 +1,2 @@
+[libfuzzer]
```

Comment:
this check needs to go in fuzz target itself

if (size > 384)
  return 0;

Reply:
That check is also in the fuzz target already.

Reply:
this is not needed here then, please remove this file.